src/xrc/src/api.rs

```diff
@@ -95,8 +95,6 @@ impl CallExchanges for CallExchangesImpl {
             return Err(CallExchangeError::NoRatesFound);
         }
 
-        rates.sort();
```

Comment:
How about sorting the rates here if `!rates.is_empty()`?

Reply:
I'm a bit confused. If the rates are no longer sorted here, where are they sorted then?

Reply:
The rates are sorted in `QueriedExchangeRate::new` [here](https://github.com/dfinity/exchange-rate-canister/blob/main/src/xrc/src/lib.rs#L533).